src/style.css

```diff
@@ -1 +1,111 @@
-/* add styles here */
+/* #region fonts and normalizations */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300px;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
```

Comment:
better:

```
html {
  box-sizing: border-box;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}

* {
  padding: 0;
  margin: 0;
  
}

```
and move these styles to reset.css and include here import './reset.css';